control.lua

```diff
@@ -93,6 +99,10 @@ local function processRequests(req, requests)
 end
 
 local function processCombinator(obj)
+    if(not buffer_chests_enabled) then
+        load_settings();
+    end
+
```

Comment:
Why is this necessary? The game should run load_settings already on load, due to the event specification you added.